cves/CVE-2007-3847.yml

```diff
@@ -201,4 +215,8 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    The date input causing the issue seems to have been trusted without doing
+    all of the proper validation and checking before using it. As mentioned
+    earlier I believe this vulnerability could have been avoided if bound
+    checking was performed.
```

Comment:
How can we prevent this kind of problem in the future?

Reply:
I will add that in my next push